tests/tools/microServer/uWServer.py

```diff
@@ -653,6 +674,15 @@ def main():
                         type=str,
                         default='',
                         help="A file which will install observers on hooks")
+    parser.add_argument("--lookupkey","-lkey",
```

Comment:
I thought `-` arguments had to be single letters.

Reply:
I guess you are right. I will fix it